pkg/flow/import_test.txtar

```diff
@@ -0,0 +1,283 @@
+-- declare_passthrough --
```

Comment:
nit: typically files like these belong in a `testdata/` subdirectory (relative to the package being tested) to clearly indicate its assets only used in testing.